src/features/nba/nbaReducer.js

```diff
@@ -0,0 +1,39 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const fetchTeams = async () => {
+  const url = 'https://www.balldontlie.io/api/v1/teams';
+  const data = await fetch(url);
+  const teams = await data.json();
+  return teams;
+};
+
+export const getTeams = createAsyncThunk(
+  'fetch/fetchTeams',
+  fetchTeams,
+);
+
+const teamsSlice = createSlice({
+  name: 'store',
+  initialState: {
+    teams: [],
+    status: 'idle',
+  },
+  reducers: {},
+  extraReducers: {
+    [getTeams.pending]: (state) => {
+      const response = state;
+      response.status = 'loading';
+    },
+    [getTeams.fulfilled]: (state, actions) => {
+      const response = state;
+      response.status = 'success';
+      response.teams = actions.payload;
+    },
+    [getTeams.rejected]: (state) => {
+      const response = state;
+      response.status = 'rejected';
+    },
+  },
+});
+
+export default teamsSlice.reducer;
```

Comment:
- [ ] For processing API calls, you are required to make use of Redux. You require to have the Redux action types, action creators and your reducer here. Good job using CreateAsyncThunk in your project now only include Redus as well.

![image](https://user-images.githubusercontent.com/86318284/203926285-ec939356-1ccf-4316-aa99-e7ef468373a8.png)